spec/support/shared/examples/msf/db_manager/migration.rb

```diff
@@ -52,35 +52,28 @@ def migrate
     end
 
     context 'with StandardError from ActiveRecord::Migration.migrate' do
-      let(:error) do
+      let(:standard_error) do
         StandardError.new(message)
       end
 
       let(:message) do
-        "Error during migration"
+        "DB.migrate threw an exception"
```

Comment:
The error message was changed slightly by https://github.com/rapid7/metasploit-framework/pull/13608 :

Line:
https://github.com/rapid7/metasploit-framework/commit/1a2bf98222cd7d1a51da459fbd86b65e5fe3d500#diff-47e1c20341699502d61cf11955a590cbL49-R49